src/lib.js

```diff
@@ -131,6 +131,38 @@ SyncBuffer.prototype.get_buffer = function(fn)
 
 (function()
 {
+    var clz32 = Math.clz32;
+    if(typeof clz32 === 'function' && clz32(0) === 32 &&
```

Comment:
Use double quotes.

Reply:
...or maybe we can just get rid of this polyfill at all?